.github/workflows/on-pull-request.yaml

```diff
@@ -0,0 +1,25 @@
+name: On Pull Request
+
+on:
+  pull_request:
+    branches: [ main ]
+  workflow_dispatch:
+
+
+jobs:
+  verify-readme:
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Verify README generation
+        uses: momentohq/standards-and-practices/github-actions/oss-readme-template@gh-actions-v1
+        with:
+          project_status: official
+          project_stability: alpha
+          project_type: other
+          sdk_language: PHP
```

Comment:
nit: I don't think this field gets used for non-SDK artifacts.  Might be more clear to remove it.